src/Utility/Vec3.h

```diff
@@ -97,6 +97,9 @@ class Vec3
     return *this;
   }
 
+  //! \brief Negation operator.
+  Vec3 operator-() const { return Vec3(-x,-y,-z); }
```

Comment:
should this go in Vec3Oper like the other operators?

Reply:
Vec3Oper only contains global operators, i.e., functions taking the Vec3 object(s) they operate on as arguments. Vec3.h contains all the other operators, like assignment, +=, indexing, etc. Think the unary operator fits well here too.